frontend/src/containers/Addgroup.js

```diff
@@ -0,0 +1,392 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+
+import LoginTopBar from '../containers/LoginTopBar';
+import Notification from '../containers/Notification';
+
+import CustomTextArea from '../components/CustomTextArea';
+import ImagePicker from '../components/ImagePicker';
+import Input from '../components/CustomTextArea';
+import PublicFooter from '../components/PublicFooter';
+import Select from '../components/Select';
+
+import appendGroupForm from '../actions/form/append_group';
+import createGroup from '../actions/groups/create';
+import notify from '../actions/notification/notify';
+import uploadImage from '../actions/images/upload';
+import validateSchema from '../actions/form/validate_schema';
+
+import groupSchema from '../joi_schemas/group';
+
+import i18n from '../lib/i18n';
+
+const memberRole = 'MEMBER';
+
+export class Addgroup extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const {
+      appendGroupForm,
+      groupForm,
+      i18n
+    } = this.props;
+
+    const {
+      name,
+      slug,
+      currency,
+      website,
+      video,
+      mission,
+      description,
+      longDescription,
+      whyJoin,
+      tags,
+      users1,
+      users2,
+      users3,
+      users4,
+      users5
+    } = groupForm.attributes;
+
+    return (
+      <div className='Login'>
+        <Notification {...this.props} />
+        <LoginTopBar />
+        <div className='Addgroup'>
+          <div className='Addgroup-header'>
+            <strong>Create a new collective</strong>
+          </div>
+            <div className="Addgroup-form-container">
+              <div className="sm-flex items-stretch">
+                <div className="order-2">
+                  <div className='Addgroup-imagepicker-cont pb3'>
+                    <ImagePicker
+                      uploadOptionFirst
+                      className="logo"
+                      dontLookupSocialMediaAvatars
+                      handleChange={logo => appendGroupForm({logo})}
+                      label="Select collective logo"
+                      uploadImage={uploadImage}
+                      i18n={i18n}
+                      presets={['/static/images/repo.svg', '/static/images/code.svg', '/static/images/rocket.svg']}
```

Comment:
I'd move the preset arrays to its own constant variable and hoist it above the class for easier future editing, also the array is used again below, so we reuse the Array instead of creating two, with the same elements.

 ```
const LOGO_PRESET = ['/static/images/repo.svg', '/static/images/code.svg', '/static/images/rocket.svg']
```
...
`presets={ LOGO_PRESETS }`
...